lib/config.js

```diff
@@ -519,6 +519,17 @@ function readEnv () {
     }
   }
 
+  // legacy keys for backward compatability
+  const legacyKeys = {
+    sanitizeFieldNames: 'ELASTIC_SANITIZE_FIELD_NAMES',
+    ignoreMessageQueues: 'ELASTIC_IGNORE_MESSAGE_QUEUES'
+  }
```

Comment:
`ENV_TABLE` already supports an array of envvars for a config var. Could we use that?

As well, the legacy ones should go first because the last one wins, and I think `ELASTIC_APM_SANITIZE_FIELD_NAMES` should win over `ELASTIC_SANITIZE_FIELD_NAMES` if both are set.

Reply:
Hah -- I thought you might object to an export for what is, effectively, a linting.  I've changed things up to use exported variables and the "list an array for multiple configuration fields" thing.